include/seqan/arg_parse/arg_parse_parse.h

```diff
@@ -307,6 +308,31 @@ ArgumentParser::ParseResult parse(ArgumentParser & me,
         return ArgumentParser::PARSE_ERROR;
     }
 
+#if _SEQAN_VERSION_CHECK == 1
+    // do version check if not turned off by the user
+    std::string check = "DEV"; // default
+    if(isSet(me, "version-check"))
+        getOptionValue(check, me, "version-check");
```

Comment:
Why getting the version check value if version-check is already set?
Can we not just use this as a flag option?
Like so:     
```
addOption(parser, seqan::ArgParseOption("", "version-check", "Check for updates."));
...
bool doVersionCheck = isSet(me, "vc");
```
And then only use doVersionCheck instead of testing against an expected string.